### PR TITLE
MEP_Engine: replace calls to GetShallowClone

### LINKED ISSUE #2557: MEP_Engine: Replace calls to Clone methods with calls to Base DeepClone or ShallowClone
#### Broken rules:
See https://github.com/BHoM/admin/issues/9

#### Suggestions to restore compliance:
https://github.com/BHoM/admin/issues/9#issuecomment-709032648

### PR DESCRIPTION
**Pull request by alelom**
### Issues addressed by this PR

Closes #2557

### Test files

Just compliance, should mean no functional change

### CODE REVIEW

**Review by FraserGreenroyd — APPROVED** [review]
LGTM

**Review by FraserGreenroyd — CHANGES_REQUESTED** [review]
This PR relates entirely to Physical_Engine, which has already been reviewed and merged in #2556 

@alelom can you check the files changed for this branch? I think there might have been an issue with changing branches to fix whatever was needed for MEP Engine?

**Comment by alelom** [comment]
> This PR relates entirely to Physical_Engine

my bad. Must've confused where I launched my Find-Replace 😄 

Should be ok now. @FraserGreenroyd

File: MEP_Engine/Modify/Transform.cs [hunk 1]
```diff
@@ -48,7 +49,7 @@ public static CableTray Transform(this CableTray cableTray, TransformMatrix tran
                 return null;
             }
 
-            CableTray result = cableTray.GetShallowClone() as CableTray;
+            CableTray result = cableTray.ShallowClone() as CableTray;
```
Comment (FraserGreenroyd) [inline MEP_Engine/Modify/Transform.cs:52]:
```suggestion
            CableTray result = cableTray.ShallowClone();
```

File: MEP_Engine/Modify/Transform.cs [hunk 2]
```diff
@@ -74,7 +75,7 @@ public static Duct Transform(this Duct duct, TransformMatrix transform, double t
                 return null;
             }
 
-            Duct result = duct.GetShallowClone() as Duct;
+            Duct result = duct.ShallowClone() as Duct;
```
Comment (FraserGreenroyd) [inline MEP_Engine/Modify/Transform.cs:78]:
```suggestion
            Duct result = duct.ShallowClone();
```

**Review by FraserGreenroyd — CHANGES_REQUESTED** [review]

**Review by FraserGreenroyd — APPROVED** [review]

**Comment by FraserGreenroyd** [comment]
@BHoMBot check required
@BHoMBot check dataset-compliance
@BHoMBot check copyright-compliance
@BHoMBot check ready-to-merge

**Comment by FraserGreenroyd** [comment]
@BHoMBot check required
@BHoMBot check dataset-compliance
@BHoMBot check copyright-compliance
@BHoMBot check ready-to-merge

**Review by FraserGreenroyd — APPROVED** [review]

**Pull request merged**